docs/platforms/apple/guides/ios/session-replay/customredact.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: Using Custom Masking for Session Replay
+sidebar_order: 5501
+notSupported:
+description: "Learn how to mask parts of your app's data in Session Replay."
+---
+
+<Alert>
+
+If you choose to use custom masking in your Session Replays, you may accidentally expose sensitive customer data. Be sure to double-check what you choose to expose.
```

Comment:
That's because for that they must disable the default masking, right?
Worth calling that out?

@lizokm might have ideas?